### PR TITLE
Mbarba/pyproject

### PR DESCRIPTION
**Pull request by MatBarba**
Pyproject file shamelessly copied from ensembl-genomio

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+[build-system]
+requires = [
+    "setuptools",
+    "setuptools-scm",
+    "wheel"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 110
```
Comment (vsitnik) [inline pyproject.toml:86]: no new line. is it as expected?

**Review by vsitnik — APPROVED** [review]
could you add a note / update one on how to install the python part to README / other docs

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+**/__pycache__
+**/*.egg-info
+**/pip-wheel-metadata
+**/build
```
Comment (MatBarba) [inline .gitignore:4]:
```suggestion
**/build

```

File: pyproject.toml (hunk 1 above)
Comment (MatBarba) [inline pyproject.toml:86]:
```suggestion
line-length = 110

```

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""setuptools based stub for the "editable" installations"""
+
+from setuptools import setup
+
+if __name__ == "__main__":
+    setup()
```
Comment (MatBarba) [inline setup.py:20]:
```suggestion
    setup()

```

File: pyproject.toml (hunk 1 above)
Comment (MatBarba) [reply pyproject.toml]: nope, now added

**Review by vsitnik — APPROVED** [review]
thanks!

**Pull request merged**